geopandas/io/tests/test_file.py

```diff
@@ -412,7 +412,8 @@ def test_infer_zipped_file():
     gdf = read_file(path)
     assert isinstance(gdf, geopandas.GeoDataFrame)
 
-    # Check that it can successfully add a zip scheme to a path that already has a scheme
```

Comment:
pylint gave an error there https://github.com/geopandas/geopandas/runs/3260865832#step:4:56

Should do this in another PR before?